vllm-tt-metal-llama3-70b/src/run_vllm_api_server.py

```diff
@@ -25,6 +30,7 @@ def get_encoded_api_key(jwt_secret):
     return encoded_jwt
 
 
+@patch.object(LLMEngine, "__init__", new=init_wrapper)
```

Comment:
the init_wrapper should be named to indicate it is for logging only, e.g. logging_init_wrapper.

Reply:
got it, it is changed now